swoole_http_request.cc

```diff
@@ -696,10 +699,6 @@ static int multipart_body_on_data_end(multipart_parser* p)
     {
         long size = swoole_file_get_size((FILE *) p->fp);
         add_assoc_long(z_multipart_header, "size", size);
-        if (size == 0)
-        {
-            add_assoc_long(z_multipart_header, "error", HTTP_UPLOAD_ERR_NO_FILE);
```

Comment:
Allow legitimate upload of empty file with zero contents not to be confused with blank form field.